rust/decentraland-godot-lib/src/dcl/js/mod.rs

```diff
@@ -206,18 +242,22 @@ pub(crate) fn scene_thread(
         .borrow_mut()
         .put(SceneStartTime(std::time::SystemTime::now()));
 
+    if inspector.is_some() {
+        // TODO: maybe send a message to announce the inspector is being waited
```

Comment:
A log just not be enought?

Reply:
Inpsector is already sending a log with the ws addres, if you have the inspector opened, it's automatically listening for new host with 9222 port, so it's not necessary, the feedback can be thought in the future for content creators debugging